### ISSUE TITLE
A little problem about training

### ISSUE THREAD

**Issue by loveXMTX** [CSAILVision/semantic-segmentation-pytorch#57]
How can I load the pretrained model to tain? I have just seen the argument '--start_epoch',if I want to use the test model 'epoch_20' you offered to train,how can I do?

**Comment by Tete-Xiao** [comment]
--start_epoch only affects the learning rate when the training starts. It is necessary for we use the "poly" learning rate policy, which you may refer to several segmentation papers or our codes for further detail.
